venia-integration-tests/update-tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0 ]; then
+    echo "Please provide URL to run tests against."
+    exit 1
+fi
+
+# collect pr instance url to run tests against
+PR_INSTANCE_URL=$1
+
+# build docker with UPDATE_SNAPSHOT=true to update failed snapshots
+docker build -f cypress-tests.dockerfile -t cypress-test --build-arg PR_INSTANCE_URL=$PR_INSTANCE_URL --build-arg UPDATE_SNAPSHOT=true .
```

Comment:
if the only difference between this script and the one above is `UPDATE_SNAPSHOT=true` then this value should just be passed in as a parameter to this script. no need to have 2 scripts doing the exact same thing.

Reply:
You are right, actually, they had different logic initially when I created 2 files. I changed the code yesterday and forgot to optimize the usage. Will do it.